test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -1941,7 +1942,11 @@ protected Settings nodeSettings(int nodeOrdinal) {
             .put(SearchService.LOW_LEVEL_CANCELLATION_SETTING.getKey(), randomBoolean())
             .putList(DISCOVERY_SEED_HOSTS_SETTING.getKey()) // empty list disables a port scan for other nodes
             .putList(DISCOVERY_SEED_PROVIDERS_SETTING.getKey(), "file")
-            .put(featureFlagSettings());
+            .put(featureFlagSettings())
+            // By default, for tests we will put the target slice count of 2. This will increase the probability of having multiple slices
+            // when tests are run with concurrent segment search enabled. When concurrent segment search is disabled then it's a no-op as
+            // slices are not used
+            .put(SearchBootstrapSettings.CONCURRENT_SEGMENT_SEARCH_TARGET_MAX_SLICE_COUNT_KEY, 2);
```

Comment:
Doesn't need to be in this PR, but I think we can consider exposing this as a function like `featureFlagSettings()` for individual tests to override.

Reply:
Individual tests can override it by overriding the `nodeSettings` method. It will first call `super.nodeSettings` and then add the test specific custom settings to the returned object and then return that.